src/components/I18nProvider.js

```diff
@@ -0,0 +1,25 @@
+import { Component } from 'preact';
+
+import getTranslations from '../utils/i18n';
+
+export default class I18nProvider extends Component {
+    constructor(props) {
+        super(props);
+
+        const { language } = this.props;
+
+        this.state = { i18n: getTranslations(language) };
+    }
+
+    getChildContext() {
+        const { i18n } = this.state;
+
+        return { i18n };
```

Comment:
Seems like this could be simplified to: `return this.state`?

Reply:
I've tried, linter throws `'i18n' in state ins not used` :/